GenderPayGap.WebUI/Models/Report/ReportFiguresViewModel.cs

```diff
@@ -63,7 +63,7 @@ public class ReportFiguresViewModel : GovUkViewModel
             NameWithinSentence = "difference in bonus pay (median)")]
         [GovUkValidateDecimalRange(Minimum = (double) decimal.MinValue, Maximum = 100)]
         [GovUkValidationRegularExpression(PositiveAndNegativeOneDecimalPlaceRegex, OneDecimalPlaceErrorMessage)]
-        [DisplayFormat(DataFormatString = "{0:0.#}")]
+        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:0.#}")]
         public decimal? DiffMedianBonusPercent { get; set; }
```

Comment:
`GovUkValidateRequired` should be removed as `DiffMedianBonusPercent` and `DiffMeanBonusPercent` are only required if `MaleBonusPayPercent` is not 0.

We already have validation rules for this in `ValidateBonusPayIntegrity`